how-to-guides/present-cross-validation-results/notebooks/Neptune_cross_validation.ipynb

```diff
@@ -0,0 +1,583 @@
+            "  Stored in directory: /root/.cache/pip/wheels/66/62/8a/356e365defb466fb97dd3bf11b45004138549cef8bdd2cf7b7\n",
+            "  Building wheel for future (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Created wheel for future: filename=future-0.18.2-py3-none-any.whl size=491070 sha256=0d5c990611f1937c91165f776640e05da5d4d4ac7d7caeba0e89582626270029\n",
+            "  Stored in directory: /root/.cache/pip/wheels/56/b0/fe/4410d17b32f1f0c3cf54cdfb2bc04d7b4b8f4ae377e2229ba0\n",
+            "  Building wheel for strict-rfc3339 (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Created wheel for strict-rfc3339: filename=strict_rfc3339-0.7-py3-none-any.whl size=18149 sha256=31689be63f2b00e2f869ba8c6cc8bce11d904f38e28fd0f0df33cbd90a07816c\n",
+            "  Stored in directory: /root/.cache/pip/wheels/f3/1d/9f/2a74caecb81b8beb9a4fbe1754203d4b7cf42ef5d39e0d2311\n",
+            "Successfully built neptune-client future strict-rfc3339\n",
+            "Installing collected packages: webcolors, urllib3, strict-rfc3339, rfc3987, jmespath, swagger-spec-validator, smmap, simplejson, jsonref, botocore, s3transfer, monotonic, gitdb, bravado-core, websocket-client, PyJWT, GitPython, future, bravado, boto3, neptune-client\n",
+            "  Attempting uninstall: urllib3\n",
+            "    Found existing installation: urllib3 1.24.3\n",
+            "    Uninstalling urllib3-1.24.3:\n",
+            "      Successfully uninstalled urllib3-1.24.3\n",
+            "  Attempting uninstall: future\n",
+            "    Found existing installation: future 0.16.0\n",
+            "    Uninstalling future-0.16.0:\n",
+            "      Successfully uninstalled future-0.16.0\n",
+            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
+            "datascience 0.10.6 requires folium==0.2.1, but you have folium 0.8.3 which is incompatible.\u001b[0m\n",
+            "Successfully installed GitPython-3.1.24 PyJWT-2.3.0 boto3-1.19.4 botocore-1.22.4 bravado-11.0.3 bravado-core-5.17.0 future-0.18.2 gitdb-4.0.9 jmespath-0.10.0 jsonref-0.2 monotonic-1.6 neptune-client-0.13.0 rfc3987-1.3.8 s3transfer-0.5.0 simplejson-3.17.5 smmap-5.0.0 strict-rfc3339-0.7 swagger-spec-validator-2.7.3 urllib3-1.25.11 webcolors-1.11.1 websocket-client-1.2.1\n"
+          ]
+        }
+      ],
+      "source": [
+        "! pip install neptune-client numpy==1.19.5 torch torchvision sklearn"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gXve6tFt1dLd"
+      },
+      "source": [
+        "**Import libraries**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "n1NVL5h6MlLq",
+        "tags": []
+      },
+      "outputs": [],
+      "source": [
+        "import neptune.new as neptune\n",
```

Comment:
make sure that you sort imports properly -> PEP 8 -- Style Guide for Python Code -> https://www.python.org/dev/peps/pep-0008/#imports